src/main/resources/templates/leftBehind.html

```diff
@@ -46,6 +46,26 @@ <h2>Left behind tasks</h2>
             <img th:src="|/img/cat_pm_${user.pussyMeter}.svg|" alt="cat">
         </div>
     </main>
+    <footer>
+        <p th:if="${deletedTask} == true"  class="fadeoutMessage" id="deletedTask"> You have completely deleted the task </p>
+        <p th:if="${reAddedTask} == true"  class="fadeoutMessage"> You have readmitted the task</p>
+    </footer>
 </section>
 </body>
-</html>
\ No newline at end of file
+</html>
+<script th:inline="javascript">

```

Comment:
`th:inline="javascript"`  is unnecessary

Reply:
are u sure about this? it was necessary for receiving a constant from backend

Reply:
prave ze me to vubec predtim nefungovalo bez nim